src/app/components/select-field/index.html

```diff
@@ -166,31 +166,32 @@
       propertyName="data"
       isOptional="true"
     >
-      An array of <a stacheRouterLink="." fragment="skyselectfield-properties">The <sky-code>SkySelectField</sky-code> objects</a> provided to the select field's <sky-code>data</sky-code> property. Use this property to populate your custom picker with options for the user to select.
+      Receives an array <a stacheRouterLink="." fragment="skyselectfield-properties">of <sky-code>SkySelectField</sky-code> objects</a> from the select field's <sky-code>data</sky-code> property. Use this property to populate the custom picker with items for users to select.
```

Comment:
I'm not sure about the "Receives" wording. It's not a term typically used by developers when describing properties on objects. I think a simple description of the property would be sufficient here (i.e. "An array of SkySelectField objects...").